index.bs

```diff
@@ -2153,6 +2154,68 @@ It is recommended that once [=explicit consent=] is granted for a specific [=/or
 
+Data adjustments {#data-adjustments-header}
+----------------
+
+In some cases, security and privacy threats can be mitigated through <dfn>data adjustment</dfn>s such as throttling, quantizing, rounding, limiting, or otherwise manipulating the data reported from the [=/XR device=]. This may sometimes be necessary to avoid fingerprinting, even in situations when [=user intent=] has been established.  However, [=data adjustment=] mitigations MUST only be used in situations which would not result in user discomfort.
+
+### Throttling ### {#throttling}
+Throttling is when [=sensitive information=] is reported at a lower frequency than otherwise possible. This mitigation has the potential to reduce a site's ability to infer user intent, location, or perform user profiling. However, when not used appropriately throttling runs a significant risk of causing user discomfort. In addition, under many circumstances it may be inadequate to provide a complete mitigation.
+
+### Rounding, quantization, and fuzzing ### {#rounding-and-friends}
+Rounding, quantization, and fuzzing are three categories of mitigations that modify the raw data that would otherwise be returned to the developer. Rounding decreases the precision of data by reducing the number of digits used to express it. Quantization constrains continuous data to instead report a discrete subset of values. Fuzzing is the introduction of slight, random errors into the the data. Collectively, these mitigations are useful to avoid fingerprinting, and are especially useful when doing so does not cause noticeable impact on user comfort.
+
+### Limiting ### {#limiting}
+Limiting is when data is reported only when it is within a specific range. For example, it is possible to comfortably limit reporting positional pose data when a user has moved beyond a specific distance away from an approved location. Care should be taken to ensure that the user experience is not negatively affected when employing this mitigation. It is often desireable to avoid a 'hard stop' at the at the end of a range as this may cause disruptive user experiences.
+
+Protected functionality {#protected-functionality}
+-----------------------
+
+The [=sensitive information=] exposed by the API can be divided into categories that share threat profiles and necessary protections against those threats.
+
+### Immersiveness ### {#protect-immersiveness}
+Users must be in control of when immersive sessions are created because the creation causes invasive changes on a user's machine. For example, starting an [=immersive session=] will engage the [=/XR device=] sensors, take over access to the device's display, and begin presentating immersive content which may terminate another application's access to the XR hardware. It may also incur significant power or performance overhead on some systems or trigger the launching of a status tray or storefront.
+
+<div class="algorithm" data-algorithm="immersive-session-allowed">
```

Comment:
Do we need a similar algorithm for inline sessions? 

For context:
- The spec has sensitive data access requirements (#sensitive-information-header and #active-and-focused-document)
- An inline session only needs to meet sensitive data access requirements if it will actually expose sensitive data. This isn't always the case, e.g. An inline session with a viewer reference space and no inputs doesn't expose sensitive data
- _User intent_ requirements for sensitive data are spelled out in #761 and #762. Inline sessions can obtain user consent mid-session, so that seems to be covered.

However, AFAICT:
- The reference space feature requirements in #762 do not include sensitive data access requirements
- The pose data requirements in #761 only partially address sensitive data access requirements
- The algorithm below addresses sensitive data access requirements, but only for immersive sessions
- There doesn't appear to be an algorithm or set of requirements that would cause them to apply for inline sessions

Does this mean there is a chance that an inline session could be created that exposed sensitive data, but did not meet all the sensitive data access requirements? 

For example, could an insecure-origin inline session be created that exposes pose data in a local reference space? Where is the algorithm step that prevents that? 

Note: It's possible that this is covered, and I missed it. I'll review again tomorrow to double check.